### DISCUSSION TITLE
Filter and View all projects?

### DISCUSSION THREAD

**Discussion by ajtmm** [marcusolsson/obsidian-projects#445] [Help] [answered]
There's two things that would make this absolutely perfect for my ADHD brain.

1. Can I search/filter by tags in the YAML through all projects? I am new to Obsidian so I might be missing something really obvious here.
2. Is there a way to see a calendar of all the projects tasks in the calendar? I'm tracking multiple client projects with a remote team, so to see deliverables for all projects on a calendar view would be awesome. 

Thank you so much!

**Comment by marcusolsson** [comment, marked as answer]
Thanks for sharing!

1. You can create tag-based projects, which includes all notes with a specific tag. However, you can't filter within a project right now. Though I think it would be a good feature to have.
2. There's no support for task lists at the moment, but it's something I'm planning to add. See #442.
